content/en/containers/kubernetes/distributions.md

```diff
@@ -334,67 +334,61 @@ agents:
 
 ## Red Hat OpenShift {#Openshift}
 
-OpenShift comes with hardened security by default (SELinux, SecurityContextConstraints), thus requiring some specific configuration:
-- Create SCC for Node Agent and Cluster Agent
-- Specific CRI socket path as OpenShift uses CRI-O container runtime
+OpenShift comes with hardened security by default with SELinux and SecurityContextConstraints (SCC), thus requiring some specific configurations:
+- Elevated SCC access for the Node Agent and Cluster Agent
 - Kubelet API certificates may not always be signed by cluster CA
 - Tolerations are required to schedule the Node Agent on `master` and `infra` nodes
 - Cluster name should be set as it cannot be retrieved automatically from cloud provider
+- *(Optional)* Setting `hostNetwork: true` in the Node Agent to allow the Agent to make requests to cloud provider metadata services (IMDS)
 
-This configuration supports OpenShift 3.11 and OpenShift 4, but works best with OpenShift 4.
+This core configuration supports OpenShift 3.11 and OpenShift 4, but works best with OpenShift 4.
+
+Additionally log collection and APM have slightly different requirements as well.
+
+The use of Unix Domain Socket (UDS) for APM and DogStatsD can work in OpenShift, however is inadvisable. As it requires additional privileged permissions and SCC access to **both** your Datadog Agent pod and your application pod. Without these your application pod can fail to deploy. Datadog recommends disabling the UDS option to avoid this, allowing the Admission Controller inject the appropriate [TCP/IP setting][7] or [Service setting][8] for APM connectivity.
```

Comment:
```suggestion
The use of Unix Domain Socket (UDS) for APM and DogStatsD can work in OpenShift. However, Datadog does not recommend this, as it requires additional privileged permissions and SCC access to **both** your Datadog Agent pod and your application pod. Without these, your application pod can fail to deploy. Datadog recommends disabling the UDS option to avoid this, allowing the Admission Controller to inject the appropriate [TCP/IP setting][7] or [Service setting][8] for APM connectivity.
```